src/pages/registration/contributors_tab/components/ContributorRow.tsx

```diff
@@ -68,8 +68,8 @@ export const ContributorRow = ({
 
   return (
     <TableRow>
```

Comment:
Må ikke nødvendigvis gjøre dette, men hvis vi vet at alle celler skal være top-aligned kunne vi definert det her i stedet for hver enkelt celle
```suggestion
    <TableRow sx={{ td: { verticalAlign: 'top' } }}>
```

Reply:
Ja, kan like gjerne gjøre det tror jeg.